index.html

```diff
@@ -60,13 +60,164 @@ <h3>I can help you build a product , feature or website Look through some of my
         
         </section>
 
+        <div class="body2">
+            <h1 class="s2h1">My Recent Works</h1>
```

Comment:
- [ ] Please also as required in the project requirements, you must stick to the design as much as possible (margins top, bottom, right, and left, font and font sizes) using the Figma Template provided:
- Please change the fonts of the titles to the fonts that are provided in the Figma design.
- The background color of the works section should be white.

<img width="156" alt="image" src="https://user-images.githubusercontent.com/77190203/164861787-5d61b0ba-69cc-4a5a-b41a-0e03a867309e.png">

Reply:
I cant add Figma all font family in my project because it's not accessible in my vs and linters read it as "Unexpected missing generic font family   font-family-no-missing-generic-family-keywor"